src/widget_impl/lTerm_containers_impl.ml

```diff
@@ -68,6 +69,13 @@ class virtual abox rc = object(self)
     self#compute_size_request;
     self#compute_allocations;
     self#queue_draw
+
+  method clear =
+    List.iter (fun child -> child.widget#set_parent None) children;
+    children <- [];
+    self#compute_size_request;
+    self#compute_allocations;
```

Comment:
The calls to `#compute_size_request` and `#compute_allocations` are currently not need since `children` is empty at this point, I wasn't sure if I could leave them out since some boxes may have custom implementations that depend on values other than `children`.